web-marketplace/src/components/organisms/RegistryLayout/RegistryLayout.Header.tsx

```diff
@@ -121,6 +122,15 @@ const RegistryLayoutHeader: React.FC = () => {
     ? headerColors[pathname]
     : theme.palette.primary.light;
 
+  const onLogOut = () => {
+    disconnect && disconnect();
```

Comment:
disconnect is actually async
we should probably fix that in `WalletContextType`